Dockerfile.konflux

```
@@ -20,6 +20,7 @@ RUN npm ci --omit=optional
 
 # Set Red Hat Openshift AI logo and links
 ENV ODH_LOGO=../images/rhoai-logo.svg
+ENV ODH_LOGO_DARK=../images/rhoai-logo.svg
```

```suggestion
ENV ODH_LOGO_DARK=../images/rhoai-logo-dark-theme.svg
```

after https://github.com/opendatahub-io/odh-dashboard/pull/3578 has been merged we should be ready to go